eks-hybrid-iam-ra-setup.sh

```diff
@@ -3,14 +3,21 @@
 exec >&2
 set -eu -o pipefail
 
-declare -r SECRETS_DIR="/.bottlerocket/rootfs/root/.aws"
+declare -r HOST_ROOTFS="/.bottlerocket/rootfs"
+declare -r SECRETS_DIR="${HOST_ROOTFS}/root/.aws"
+declare -r SIGNING_HELPER_SERVICE_TEMPLATE_PATH="/usr/share/aws_signing_helper_update_service.in"
+declare -r SIGNING_HELPER_SERVICE="aws_signing_helper_update.service"
```

Comment:
I would prefer not to have underscores in any unit's name. It's annoying to have 99.9% of units use kebab case and one outlier that uses snake case.
```suggestion
declare -r SIGNING_HELPER_SERVICE="aws-signing-helper-update.service"
```

Reply:
Fixed, also changed the template's file name